### ISSUE TITLE
Ubuntu package builds are out of date

### ISSUE THREAD

**Issue by itaysk** [iovisor/bcc#2678]
Im using Ubuntu Bionic, Following instructions from the INSTALL.md, I get old versions from both the stable and the nightly channels (0.10 and 0.11 respectively).
The sources confirm, e.g for nightly last updated at July 2019: https://repo.iovisor.org/apt/bionic/dists/bionic-nightly/main/binary-amd64/

**Comment by mwuertinger** [comment]
Duplicates #2676

**Comment by pjsg** [comment]
I don't think that this is a duplicate or #2676 as that refers to the Ubuntu supplied packages. This bug is about the builds that are installed by

```
echo "deb [trusted=yes] https://repo.iovisor.org/apt/xenial xenial-nightly main" | sudo tee /etc/apt/sources.list.d/iovisor.list
sudo apt-get update
sudo apt-get install bcc-tools libbcc-examples linux-headers-$(uname -r)
```
(replace xenial with artful or bionic as appropriate)

It appears that the CI builds of these stopped working back in the summer -- the most recent working build commit was 6836168.

**Comment by itaysk** [comment]
I'd like to offer my help fixing this issue. can anyone point me to where this release is defined? I've looked at travis and github actions workflows which doesn't seem related to release
@yonghong-song @mwuerting

**Comment by lorddoskias** [comment]
I can confirm that the nightly packages for ubuntu xenial are also out of date. They are based off of 6836168 which over 10 months old ...

**Comment by hadret** [comment]
For anyone interested, I'm trying to maintain relatively new versions of bunch of BPF-related projects back to Ubuntu 18.04 (bionic) and newer. Here they are:

* [bpfcc](https://launchpad.net/~hadret/+archive/ubuntu/bpfcc)
* [bpftrace](https://launchpad.net/~hadret/+archive/ubuntu/bpftrace)
* [libbpf](https://launchpad.net/~hadret/+archive/ubuntu/libbpf)

I'm basing the builds on the official Debian/Ubuntu packages.

**Comment by yonghong-song** [comment]
@hadret Thanks a lot!

**Comment by dilfish** [comment]
@hadret Great Job!

**Comment by mingregister** [comment]
> For anyone interested, I'm trying to maintain relatively new versions of bunch of BPF-related projects back to Ubuntu 18.04 (bionic) and newer. Here they are:
> 
> * [bpfcc](https://launchpad.net/~hadret/+archive/ubuntu/bpfcc)
> * [bpftrace](https://launchpad.net/~hadret/+archive/ubuntu/bpftrace)
> * [libbpf](https://launchpad.net/~hadret/+archive/ubuntu/libbpf)
> 
> I'm basing the builds on the official Debian/Ubuntu packages.

Is there any suggestion for me to install specific version? thanks.

**Comment by hadret** [comment]
> Is there any suggestion for me to install specific version? thanks.

Not really, no. I'm trying to follow latest releases and that's essentially it.

**Comment by serkan-ozal** [comment]
Hi, 

Thanks a lot @hadret 👍 

Is there any official iovisor apt-repository that I can install latest ones?

**Comment by hadret** [comment]
@serkan-ozal I believe they do provide the instructions in their readme, here: https://github.com/iovisor/bcc/blob/master/INSTALL.md#ubuntu---binary

With that said, you might be better off using official packages at this point: https://packages.ubuntu.com/search?keywords=bpfcc Or with backporting newer versions from Debian: https://packages.debian.org/search?keywords=bpfcc

Otherwise, there are my PPAs I mentioned above, but they aren't official.

**Comment by SamYuan1990** [comment]
hi team, is there any news for release a latest version?
